tests/integration/targets/aci_filter_entry/tasks/main.yml

```diff
@@ -147,6 +151,7 @@
       - 'present_missing_param.msg == "state is present but all of the following are missing: entry"'
       - present_incompatible_params is failed
       - present_incompatible_params.msg.startswith("Parameter")
+  when: query_cloud.current == []
```

Comment:
Can you split this assert so that you run what can be tested on both separatly that what should only be tested on the non-cloud one?